Sources/TripKit/categories/NSNumber+Formatter.swift

```diff
@@ -18,22 +18,48 @@ extension NSNumber {
     formatter.roundingMode = .up
     return formatter
   }()
+    
+  /// formats the NSNumber to a readable currency formatted string that auto handles the decimal places.
+  public func toMoneyString(currencyCode: String) -> String {
+    let locale = Locale.availableIdentifiers
+      .map { Locale(identifier: $0) }
+      .first { $0.currencyCode == currencyCode }
```

Comment:
We can't do this. 

This will for example pick German locale for EUR and then format all numbers using German formatting, which, would confuse many people as Germans format 3 Euro as "3,00 €" (note the comma), while most other locale would expect this as "€3.00" instead. (For USD it picks [Navajo](https://en.wikipedia.org/wiki/Navajo_language).)

I've added tests which demonstrate that.

Reply:
I see, will change this into something specific.

Reply:
removed locale handling based on currency code and left out formatter without specified decimal places which fixes this. 

Business rules wise -  are we concerned on formatting the decimal formats to the user's locale or are we adapting it to just the default currency code format? 

Tests would fail when user locale is German since:
price = 300
currency = EUR

will be `3,00 €`

currency =  USD

will be `3,00 $`

curreny = JPY

will be `300 ¥`

but this follows their locale style

Reply:
I've updated the tests to include the locale. Please check.